### PR TITLE
Use explicit casts instead of function invocation

### PR DESCRIPTION
**Pull request by SoftCreatR**
Since there's no base used for conversion, we can safely switch from function invocation to explicit casts, which can increase the performance greatly. This also saves us 4 bytes per replacement.

### CODE REVIEW

**Comment by TimWolla** [comment]
> which can increase the performance greatly

Do you have any numbers for common requests (e.g. accessing a Thread)?

> This also saves us 4 bytes per replacement.

4 bytes of what? Memory Usage? Disk usage?

**Comment by TimWolla** [comment]
In any case such a large-scale replacement is not acceptable for stable branches (5.5) and needs to target master.

File: wcfsetup/install/files/lib/util/DateUtil.class.php [hunk 1]
```diff
@@ -492,7 +492,7 @@ public static function validateDate($date)
     public static function getFirstDayOfTheWeek()
     {
         if (self::$firstDayOfTheWeek === null) {
-            self::$firstDayOfTheWeek = \intval(WCF::getLanguage()->get('wcf.date.firstDayOfTheWeek'));
+            self::$firstDayOfTheWeek = (int)WCF::getLanguage(->get('wcf.date.firstDayOfTheWeek'));
```
Comment (TimWolla) [inline wcfsetup/install/files/lib/util/DateUtil.class.php:495]: Did you perform the replacement using a naïve regular expression or with a proper tool (e.g. Rector)? If this is a naïve regular expression, then there is a great chance that an incorrect replacement introduces bugs, not just obvious syntax errors.

Comment (SoftCreatR) [reply wcfsetup/install/files/lib/util/DateUtil.class.php:495]: Did most of it by hand. Just missed that one.

**Comment by SoftCreatR** [comment]
> In any case such a large-scale replacement is not acceptable for stable branches (5.5) and needs to target master.

Found a bug anyways. I'll re-create this for master.

> Do you have any numbers for common requests (e.g. accessing a Thread)?

Not really, just pretty simple benchmarks.

> 4 bytes of what? Memory Usage? Disk usage?

Disk. Of course, this is not really a significant improvement. At least the conversion would now be consistent.

File: wcfsetup/install/files/lib/util/DateUtil.class.php (hunk 1 above)
Comment (TimWolla) [reply wcfsetup/install/files/lib/util/DateUtil.class.php:495]: Before you spend more (manual) time on this: I'm not really sold on this being an overall improvement. As an example the lack of parentheses makes precedence much less clear and `\intval()` is not really in super hot paths anyway. When I last checked with XDebug's profiler, `intval` was not anywhere near the expensive locations.

**Pull request closed without merging**